meta-lmp-base/recipes-containers/skopeo/skopeo/0002-dont-store-signatures-if-none-is-present.patch

```diff
@@ -0,0 +1,42 @@
+From 35423f29e270bc63b788e6b70c736fbd1467dad1 Mon Sep 17 00:00:00 2001
+From: Mike Sul <mike.sul@foundries.io>
+Date: Thu, 15 Jun 2023 13:43:51 +0200
+Subject: [PATCH] Don't store signatures if none is present
+
+Currently, the copy command prints the message 'Storing signatures' and
+calls the signature storing function, even if there are no signatures
+present. This can mislead users and make them believe that there are
+image signatures.
+
+The proposed change modifies the copy function to print the message and
+invoke the image storing function only if there is at least one signature.
+
+Upstream-Status: Upstream-Status: Pending
```

Comment:
Fine to drop one "Upstream-Status":

`Upstream-Status: Pending`

only

Reply:
sure, done.